packages/@aws-cdk-testing/cli-integ/tests/cli-integ-tests/bootstrapping.integtest.ts

```diff
@@ -196,6 +197,41 @@ integTest('can dump the template, modify and use it to deploy a custom bootstrap
   });
 }));
 
+integTest('a customized template vendor will not overwrite the default template', withoutBootstrap(async (fixture) => {
```

Comment:
I am being thick as porridge today, should the test not be the other way around based on
>To ensure that your customizations are not accidentally overwritten later by someone running `cdk bootstrap` using the default template.
?